tasks/page/makeReadme.js

```diff
@@ -53,10 +62,24 @@ if (require.main !== module) {
   )
 }
 
-function escapeRegExp(string) {
-  return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&') // $& means the whole matched string
-}
-
 function addSpacing(string) {
   return string.replace(/[^ `](`[^\`].*?`)/g, ' $1')
 }
+
+// replace blob image urls with raw image urls, they don't work outside of github
+function replaceImageUrls(string) {
+  const imageUrl = /(https:\/\/github\.com\/.*?\/)(blob)(\/master\/.*?.(:?png|jpeg|jpg|gif|bmp))/gi
+
+  return string.replace(imageUrl, '$1raw$3')
+}
+
```

Comment:
Can we have comment here explaining this please.

Reply:
Done.